app/frontend/courses/review/components/SubmissionChecklistItemShow.res

```diff
@@ -127,15 +127,17 @@ let make = (~index, ~checklistItem, ~updateChecklistCB, ~checklist) => {
   let status = ChecklistItem.status(checklistItem)
 
   <div ariaLabel={ChecklistItem.title(checklistItem)}>
-    <div className="text-sm font-medium flex items-center justify-between ">
-      <div className="flex flex-1 items-start space-x-2">
-        <div className="flex items-start">
+    <div className="text-sm font-medium justify-between ">
+      <div className="flex items-start space-x-2">
+        <div className="flex items-start w-full gap-3">
           <PfIcon className={kindIconClasses(ChecklistItem.result(checklistItem))} />
-          <MarkdownBlock
-            profile=Markdown.Permissive
-            markdown={ChecklistItem.title(checklistItem)}
-            className="ms-3 pe-2 tracking-wide"
-          />
+          <div className="overflow-x-scroll">
```

Comment:
```suggestion
          <div className="overflow-x-auto">
```

The "overflow-scroll" property is currently displaying a default scroll bar for each question.
<img width="351" alt="Screenshot 2023-07-09 at 11 01 38 PM" src="https://github.com/pupilfirst/pupilfirst/assets/16238306/ec8affe9-af03-492b-9c81-105e7a2e00de">